pkg/logs/input/docker/launcher.go

```diff
@@ -62,7 +62,7 @@ func NewLauncher(sources *config.LogSources, services *service.Services, pipelin
 	// a channel that will lock the scheduler in case of setup failure
 	// FIXME(achntrl): Find a better way of choosing the right launcher
 	// between Docker and Kubernetes
-	launcher.sources = sources.GetSourceStreamForType(config.DockerType)
+	launcher.sources = sources.GetAddedForType(config.DockerType)
```

Comment:
Should we have a `sources.GetRemovedForType(config.DockerType)`?

Reply:
Agreed, as discussed IRL, we should open a new PR for this, thanks for finding this out.

Reply:
👍